R/qc_functions.R

```diff
@@ -0,0 +1,353 @@
+#' @importFrom stats qnorm quantile cor lm predict sd
+#' @importFrom dplyr %>% group_by summarise mutate select n filter pull arrange
+#' @importFrom graphics hist par text plot.new
+#' @importFrom grDevices dev.new
+#' @importFrom rlang .data
+#' @importFrom utils read.table write.table
+NULL
+
+# Declare global variables to avoid R CMD check notes
+utils::globalVariables(c("FINNGENID", "fixed_slope"))
+
+#' @title Quantile Normalize Values
+#' @description Performs quantile normalization on a numeric vector
+#' @param x Numeric vector to normalize
+#' @return Quantile normalized vector with same length as input
+#' @export
+quantile_normalize <- function(x) {
```

Comment:
just semantics but this is inverse rank normalization. quantile normalisation is making 2 distributions equal. consider changing name of the function